docs/content/preview/releases/release-notes/v2.12.md

```diff
@@ -17,6 +17,68 @@ Included here are the release notes for all releases in the v2.12 release series
 
 For an RSS feed of the release notes for all release series, point your feed reader to the [RSS feed for releases](../../index.xml).
 
+## v2.12.12.0 - February 10, 2023 {#v2.12.12.0}
+
+**Build:** `2.12.12.0-b12`
+
+**Third-party licenses:** [YugabyteDB](https://downloads.yugabyte.com/releases/2.12.12.0/yugabytedb-2.12.12.0-b12-third-party-licenses.html), [YugabyteDB Anywhere](https://downloads.yugabyte.com/releases/2.12.12.0/yugabytedb-anywhere-2.12.12.0-b12-third-party-licenses.html)
+
+### Downloads
+
+<ul class="nav yb-pills">
+  <li>
+    <a href="https://downloads.yugabyte.com/releases/2.12.12.0/yugabyte-2.12.12.0-b12-darwin-x86_64.tar.gz">
+      <i class="fa-brands fa-apple"></i><span>macOS</span>
+    </a>
+  </li>
+  <li>
+    <a href="https://downloads.yugabyte.com/releases/2.12.12.0/yugabyte-2.12.12.0-b12-linux-x86_64.tar.gz">
+      <i class="fa-brands fa-linux"></i><span>Linux x86</span>
+    </a>
+  </li>
+  <li>
+    <a href="https://downloads.yugabyte.com/releases/2.12.12.0/yugabyte-2.12.12.0-b12-el8-aarch64.tar.gz">
+      <i class="fa-brands fa-linux"></i><span>Linux ARM</span>
+    </a>
+  </li>
+</ul>
+
+### Docker
+
+```sh
+docker pull yugabytedb/yugabyte:2.12.12.0-b12
+```
+
+### Improvements
+
+#### Yugabyte Platform
+
+[PLAT-6305] Upgrade Golang third party vulnerabilities (azcopy version to 10.16.2)
+[PLAT-6358] Allow defining prometheus remote write config in helm chart
+
+#### Database
+
+[[15849](https://github.com/yugabyte/yugabyte-db/issues/15849)] DocDB: Don't hold mutex when starting TransactionStatusResolver
+
+### Bug fixes
+
+#### Yugabyte Platform
+
+[PLAT-4654] Edit primary cluster of kubernetes can delete the primary cluster.
+[PLAT-4598] Fix simultaneous scaling down and instance type changes in edit universe.
+[PLAT-6755] [PLAT-6756] Platform comes out from the universe page after pausing and rolling restart of universe.
+[PLAT-6834] Fix three master server graphs
+[PLAT-7338] Live queries failing on universe with custom ports.
+
+#### Database
+
+[[13891](https://github.com/yugabyte/yugabyte-db/issues/13891)] Do not update index permissions when table is being deleted
+[[14417](https://github.com/yugabyte/yugabyte-db/issues/14417)] [DocDB] YB-Tserver Registration hazard - UUID can be blank
+[[15172](https://github.com/yugabyte/yugabyte-db/issues/15172)] [YSQL] Move the ybRunContext to be a child of TopPortalContext
+[[15698](https://github.com/yugabyte/yugabyte-db/issues/15698)] [DocDB] Handle null placement info due to faulty restores
+[[15820](https://github.com/yugabyte/yugabyte-db/issues/15820)] [YSQL] Fix ybgin index scan after DROP COLUMN
+[[15863](https://github.com/yugabyte/yugabyte-db/issues/15863)] Fixed race in Tablet::FlushWriteIndexBatch
+
```

Comment:
Should have known issues in here, too. (Looks like I forget them in some others!)
```suggestion

### Known issues

* N/A

```